src/fromager/resolver.py

```diff
@@ -37,59 +34,6 @@
 SUPPORTED_TAGS = set(sys_tags())
 
 
-class Candidate:
```

Comment:
Had to shift this class to a separate module so that extras_provider can use it. Otherwise it was throwing a circular import error